src/main/drivers/pwm_mapping.c

```diff
@@ -129,10 +129,16 @@ pwmOutputConfiguration_t *pwmInit(drv_pwm_config_t *init)
 #endif
 
 #if defined(STM32F303xC) && defined(USE_UART3)
-        // skip UART3 ports (PB10/PB11)
+#if defined(AVOID_UART3_FOR_PWM_PPM)
+        if (!init->usePPM)
+            // skip UART3 ports (PB10/PB11)
+            if (init->useUART3 && (timerHardwarePtr->tag == IO_TAG(UART3_TX_PIN) || timerHardwarePtr->tag == IO_TAG(UART3_RX_PIN)))
```

Comment:
One should be able to turn off the UART in the ports page, and it will not be initialised, and thus the PIN is available for PPM so this shouldn't be necessary. Perhaps those with trouble have forgotten to turn off the UART?